docs.html

```diff
@@ -1105,6 +1122,95 @@ <h1 class="fs-l">Rating</h1>
 
             </section>
 
+            <!-- ----- Grids ------  -->
+            <section class="section" id="grid">
+                <h1 class="fs-l">Grids</h1>
+                <p class="fs-m">Grids can be used for layouting, when working in sections.</p>
+                <h2 class="fs-m">
+                    Grid two column layout (50-50 layout)
+                </h2>
+                <div class="grid-two-layout">
+                    <div class="grid-item">
+                        <p class="fs-m text-align-center">Grid Item 1</p>
+                    </div>
+                    <div class="grid-item">
+                        <p class="fs-m text-align-center">Grid Item 2</p>
+                    </div>
+                </div>
+
+                <script type="text/plain" class="language-markup line-numbers">
+                    <div class="grid-two-layout">
+                        <div class="grid-item">
+                            <p class="fs-m text-align-center">Grid Item 1</p>
+                        </div>
+
+                        <div class="grid-item">
+                            <p class="fs-m text-align-center">Grid Item 2</p>
+                        </div>
+                    </div>
+                </script>
+
+                <h2 class="fs-m">Grid three column layout</h2>
+                <div class="grid-three-layout">
+                    <div class="grid-item">
+                        <p class="fs-m text-align-center">Grid Item 1</p>
+                    </div>
+                    <div class="grid-item">
+                        <p class="fs-m text-align-center">Grid Item 2</p>
+                    </div>
+                    <div class="grid-item">
+                        <p class="fs-m text-align-center">Grid Item 3</p>
+                    </div>
+                </div>
+
+                <script type="text/plain" class="language-markup line-numbers">
+                    <div class="grid-three-layout">
+                        <div class="grid-item">
+                            <p class="fs-m text-align-center">Grid Item 1</p>
+                        </div>
+                        <div class="grid-item">
+                            <p class="fs-m text-align-center">Grid Item 2</p>
+                        </div>
+                        <div class="grid-item">
+                            <p class="fs-m text-align-center">Grid Item 3</p>
+                        </div>
+                    </div>
+                </script>
+            </section>
+
+            <!-- ------ Slider ------  -->
```

Comment:
Nice, you have used semantic HTML and maintained the proper space with comments.